modules/account-lib/src/coin/cspr/transactionBuilderFactory.ts

```diff
@@ -31,7 +33,14 @@ export class TransactionBuilderFactory extends BaseTransactionBuilderFactory {
     if (tx.casperTx.session.isTransfer()) {
       return this.getTransferBuilder(tx);
     } else if (tx.casperTx.session.isModuleBytes()) {
-      // TODO(stlx-1458), identify wallet initialization deploy
+      const moduleBytes = tx.casperTx.session.getArgByName('moduleBytes');
+      if (moduleBytes) {
+        const contract = Uint8Array.from(Buffer.from(walletInitContractHexCode, 'hex'));
```

Comment:
what's the purpose of the `contract` const? It's assigned but never used

Reply:
Oh... Thanks for noticing that.
The Idea was to use that contract variable to check if the moduleBytes session is a Wallet Initialization Transaction or not.

I'll fix it and update this PR.